### PR TITLE
Set ev.target.files empty after every change

### PR DESCRIPTION
**Pull request by pusztaienike**
If you select a file for upload it will be stored in ev.target.files, but if you cancel the upload and select the same file again onChange won't be called [- cause your input will be the same as in ev.target.files-] . 
My solution was to set event.target.value to an empty string which set the event.target.files empty as well.

### CODE REVIEW

File: apps/sensenet/src/components/dialogs/upload/upload-dialog.tsx [hunk 1]
```diff
@@ -211,7 +211,10 @@ export function UploadDialog(props: UploadDialogProps) {
         </Grid>
       </DialogContent>
       <input
-        onChange={ev => ev.target.files && addFiles([...ev.target.files])}
+        onChange={ev => {
+          ev.target.files && addFiles([...ev.target.files])
+          ev.target.value = ''
```
Comment (B3zo0) [inline apps/sensenet/src/components/dialogs/upload/upload-dialog.tsx]: I would rather see this fix in the removeItem method. Set `inputFile` value field and add a comment please that states why was this necessary.

**Comment by pusztaienike** [comment]
This closes #491
